.gitlab-ci.yml

```diff
@@ -260,6 +260,18 @@ windows32:
   except:
     - /^pr-.*$/
 
+lint:
+  image: docker:git
```

Comment:
Why to use a docker image here instead of the base one?

Reply:
I don't remember how I picked it and don't want to search for images again.

Reply:
>I mean, why do you use an image at all?

I have no clue what happens if you don't use an image. Is it even possible to unset our toplevel `image: "$IMAGE"`?

Reply:
Yeah, my comment was not very useful, your choice is fine.